tests/validation_tests/test_utils.py

```diff
@@ -45,3 +51,43 @@ def test_get_project_number_by_id():
 
     with pytest.raises(ValueError):
         get_project_number_by_id("TD-ABC-02", MOCK_ACTIVE_PROJECT_IDS)
+
+
+def test_is_numeric():
+    assert is_numeric("1")
+    assert is_numeric("10000")
+    assert is_numeric("99999999999")
+    assert is_numeric("0")
+    assert is_numeric("1.1")
+    assert is_numeric("99999999999.9")
+    assert is_numeric("0.9")
+    assert not is_numeric("0-9")
+    assert not is_numeric("0,9")
+    assert not is_numeric("0'9")
+    assert not is_numeric("0@9")
+    assert not is_numeric("zero")
+    assert not is_numeric("nine")
```

Comment:
If we think negative numbers are possibility, add a case with negative numbers to this test

Reply:
added 👍